Formula/infrakit.rb

```diff
@@ -0,0 +1,30 @@
+class Infrakit < Formula
+  desc "Toolkit for creating and managing declarative infrastructure"
+  homepage "https://github.com/docker/infrakit"
+  url "https://github.com/docker/infrakit.git",
+      :tag => "v0.5",
+      :revision => "3d2670e484176ce474d4b3d171994ceea7054c02"
+  head "https://github.com/docker/infrakit.git"
+
+  depends_on "automake" => :build
+  depends_on "go" => :build
+  depends_on "libvirt" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/docker/infrakit").install buildpath.children
+    cd "src/github.com/docker/infrakit" do
+      system "make", "cli"
+      bin.install "build/infrakit"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    ENV["INFRAKIT_DIR"] = ENV["HOME"]
```

Comment:
Just set these to `buildpath`.

Reply:
Unfortunately that will fail the test:
```sh
$ brew test infrakit
Testing infrakit
==> /usr/local/Cellar/infrakit/0.5/bin/infrakit version
panic: mkdir /Users/seb/.infrakit: operation not permitted

goroutine 1 [running]:
main.main()
	/private/tmp/infrakit-20180809-46262-6zai71/src/github.com/docker/infrakit/cmd/infrakit/main.go:46 +0xa3a
Error: infrakit: failed
<0> expected but was
<2>.
```
I've tried setting `$HOME` to `buildpath` with the same result.

Reply:
Oh shoot, I meant `testpath`, heh. Does `testpath` work?

Reply:
Thanks. Yea that works!